plugins/woocommerce/client/blocks/assets/js/base/components/cart-checkout/totals/coupon/style.scss

```diff
@@ -34,6 +34,16 @@
 		padding-right: $gap;
 		white-space: nowrap;
 		flex: 1 1 auto;
+
+		// Hides labels when the spinner is shown.
+		&--loading {
+			.wc-block-components-button__text {
+				visibility: hidden;
+			}
+			.wc-block-components-spinner {
+				visibility: visible;
+			}
```

Comment:
Do we need to do this in addition to the `{ isLoading && <Spinner /> }` that we have in the component?

Reply:
Yes, because the parent has `visibility: hidden` (which hides the text).